specs/infrastructure/05_ablation_study.md

```
@@ -0,0 +1,224 @@
+# CMS Multi-Scale Ablation Study
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)